assets/css/_ui_kit.scss

```diff
@@ -254,6 +254,26 @@ $color-tooltip-background: #333333;
   }
 }
 
+@mixin button-primary-v2 {
+  color: $white;
+  background-color: $color-eggplant-700;
+  border-radius: 4px;
+
+  &:hover {
+    background-color: $color-eggplant-500;
+  }
+
+  &:focus {
+    background-color: $color-eggplant-500;
+    outline: 2px solid $color-eggplant-700;
+    outline-offset: -2px;
```

Comment:
This didn't affect the button size in the way that adding a border did (even with `box-sizing: border-box` - maybe something small I'm missing?), but open to other suggestions if there is a better approach!

This doesn't respect the border-radius for [firefox versions < 88](https://developer.mozilla.org/en-US/docs/Web/CSS/outline). Our docs say we support versions 78+, but in FS I see that the lowest version used is 112. If outline does seem like the right approach for this, wondering if it would be reasonable to update our docs to support version 88+

Reply:
I believe `outline` does not affect layout so it makes sense that it does not do what border did to the sizing https://developer.mozilla.org/en-US/docs/Web/CSS/outline#description

Reply:
If design didn't explicitly ask for the offset to be within the button, then I think it'd be better to leave it as default and remove the offset, but otherwise I think this is good!